### ISSUE TITLE
@cmd is exported but undocumented

### ISSUE THREAD

**Issue by stevengj** [JuliaLang/julia#19343]
`@cmd "some string"` is exported but there is no docstring.  (This parses a string and turns it into a `Cmd` object.)

**Comment by JeffreySarnoff** [comment]
Please resolve this by including a docstring rather than by ceasing :haircut_man: exportation of @cmd  .

**Comment by tkelman** [comment]
seems redundant with interpolating into backticks?

**Comment by stevengj** [comment]
Does interpolating call `shell_parse`?  Also, with `@cmd`, the parsing is done at parse-time rather than runtime.   I know that the REPL calls `@cmd` for processing `; ...` lines...

**Comment by JeffreySarnoff** [comment]
I did not know that @cmd parses at parse-time.  Given that, I am less concerned.

**Comment by inkydragon** [comment]
This issue has been fixed in version 1.6.4+.

```julia
help?> @cmd " echo 1 "
  @cmd str

  Similar to cmd, generate a Cmd from the str string which represents the shell command(s) to be executed. The
  Cmd object can be run as a process and can outlive the spawning julia process (see Cmd for more).

  Examples
  ≡≡≡≡≡≡≡≡≡≡

  julia> cm = @cmd " echo 1 "
  `echo 1`

  julia> run(cm)
  1
  Process(`echo 1`, ProcessExited(0))

julia> versioninfo()
Julia Version 1.6.4
Commit 35f0c911f4 (2021-11-19 03:54 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: Intel(R) Core(TM) i5-9400F CPU @ 2.90GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-11.0.1 (ORCJIT, skylake)
```
